app/views/schools/on_boarding/descriptions/_form.html.erb

```diff
@@ -13,7 +13,7 @@
   <div class="govuk-grid-column-two-thirds">
     <%= form_for description, url: schools_on_boarding_description_path, method: method do |f| %>
       <h1 class="govuk-heading-l">
-        Enter a short description of your school
+        Write a short description of your school
```

Comment:
See previous comment about this page (we should make the appropriate content changes along with the other 'guidance for schools' recommendations, with dev help).